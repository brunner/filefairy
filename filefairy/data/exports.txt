31  11  1   2   1 1 1 1 1 1 1 0 1 1
32  7   5   1   1 1 1 0 0 0 1 0 1 0
33  5   7   2   0 0 0 0 0 0 1 0 1 1
34  12  0   12  1 1 1 1 1 1 1 1 1 1
35  12  0   12  1 1 1 1 1 1 1 1 1 1
36  5   7   4   0 1 0 0 1 1 0 0 0 0
37  12  0   12  1 1 1 1 1 1 1 1 1 1
38  11  1   8   1 0 1 1 1 1 1 1 1 1
39  11  1   1   1 1 1 1 1 1 1 1 1 0
40  12  0   12  1 1 1 1 1 1 1 1 1 1
41  6   6   1   1 0 1 0 0 0 1 0 1 0
42  12  0   12  1 1 1 1 1 1 1 1 1 1
43  11  1   10  1 1 1 1 1 1 1 1 1 1
44  12  0   12  1 1 1 1 1 1 1 1 1 1
45  9   3   2   1 1 0 0 1 1 1 0 1 1
46  7   5   1   1 0 1 0 1 0 0 1 1 0
47  12  0   12  1 1 1 1 1 1 1 1 1 1
48  12  0   12  1 1 1 1 1 1 1 1 1 1
49  12  0   12  1 1 1 1 1 1 1 1 1 1
50  10  2   3   1 1 1 1 0 1 0 1 1 1
51  10  2   1   1 1 1 1 1 0 1 1 1 0
52  12  0   12  1 1 1 1 1 1 1 1 1 1
53  9   3   1   1 1 0 1 0 1 1 1 1 0
54  9   3   6   1 0 0 0 1 1 1 1 1 1
55  9   3   1   0 0 1 1 1 1 1 1 1 0
56  11  1   6   1 1 1 0 1 1 1 1 1 1
57  10  2   1   1 1 1 0 1 1 1 1 1 0
58  11  1   9   0 1 1 1 1 1 1 1 1 1
59  12  0   12  1 1 1 1 1 1 1 1 1 1
60  9   3   2   1 1 1 1 1 0 1 0 1 1
