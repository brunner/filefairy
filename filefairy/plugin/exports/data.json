{
  "ai": [],
  "date": "2018-05-31T23:56:49.412148",
  "form": {
    "31": "oooonnnnno",
    "32": "nnonooonnn",
    "33": "nnnnonnono",
    "34": "nonnonnnnn",
    "35": "onnonnnnon",
    "36": "nnnnnnnnnn",
    "37": "nonnnnnnnn",
    "38": "nooononnon",
    "39": "nnoonnnnnn",
    "40": "nnnnnnnnnn",
    "41": "noonoooono",
    "42": "nnnonnnnnn",
    "43": "onnoonoonn",
    "44": "oooonononn",
    "45": "ooooonnoon",
    "46": "nonnnnnnnn",
    "47": "nnnnnnnnnn",
    "48": "ononnnnnno",
    "49": "onnnnnnnnn",
    "50": "nnnnnonnnn",
    "51": "nnnn",
    "52": "noonnnnoon",
    "53": "onnonnnonn",
    "54": "nnnnnnnnnn",
    "55": "ooonnononn",
    "56": "oonnnnnoon",
    "57": "noooonnnnn",
    "58": "nnnnnnnnnn",
    "59": "onnnonnnnn",
    "60": "nooonnnnnn"
  },
  "locked": false
}
