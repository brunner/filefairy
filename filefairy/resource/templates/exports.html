{% extends "base.html" %}
{% import "common.html" as common %}

{% set items = [('tracker', 'Tracker'), ('recorded', 'Recorded')] %}
{% block nav %}
{{ common.nav(items) }}
{% endblock %}

{% block content %}

<!-- ^(exports.tracker) -->
<div class="container anchor" id="tracker">

{{ common.heading('cloudy', 'Tracker') }}

<!-- ^(exports.row) -->
<div class="row">

<!-- ^(exports.column) -->
<div class="col-12">

{{ common.card(live) }}

</div>
<!-- $(exports.column) -->

</div>
<!-- $(exports.row) -->

</div>
<!-- $(exports.tracker) -->

<!-- ^(exports.recorded) -->
<div class="container anchor" id="recorded">

{{ common.heading('target', 'Recorded') }}

<!-- ^(exports.row) -->
<div class="row">
{% set n = standings|length() // 2 %}

<!-- ^(exports.column) -->
<div class="col-md-6">

{% for table in standings[:n] %}
{{ common.table(table) }}

{% endfor %}
</div>
<!-- $(exports.column) -->

<!-- ^(exports.column) -->
<div class="col-md-6">

{% for table in standings[n:] %}
{{ common.table(table) }}

{% endfor %}
</div>
<!-- $(exports.column) -->

</div>
<!-- $(exports.row) -->

</div>
<!-- $(exports.recorded) -->

{% endblock %}
