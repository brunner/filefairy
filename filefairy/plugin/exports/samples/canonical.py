#!/usr/bin/env python3
# -*- coding: utf-8 -*-

import os
import re
import sys

_path = os.path.dirname(os.path.abspath(__file__))
_root = re.sub(r'/plugin/exports/samples', '', _path)
sys.path.append(_root)
from plugin.exports.exports import Exports  # noqa
from util.component.component import card  # noqa
from util.component.component import table  # noqa
from util.team.team import logo_absolute  # noqa

_t31 = [logo_absolute('31', 'Arizona', 'left'), '5 - 5', 'W1']
_t32 = [logo_absolute('32', 'Atlanta', 'left'), '10 - 0', 'W10']
_t33 = [logo_absolute('33', 'Baltimore', 'left'), '6 - 4', 'W1']
_t34 = [logo_absolute('34', 'Boston', 'left'), '10 - 0', 'W10']
_t35 = [logo_absolute('35', 'Chicago', 'left'), '9 - 1', 'L1']
_t36 = [logo_absolute('36', 'Chicago', 'left'), '8 - 2', 'L1']
_t37 = [logo_absolute('37', 'Cincinnati', 'left'), '10 - 0', 'W10']
_t38 = [logo_absolute('38', 'Cleveland', 'left'), '8 - 2', 'W2']
_t39 = [logo_absolute('39', 'Colorado', 'left'), '10 - 0', 'W10']
_t40 = [logo_absolute('40', 'Detroit', 'left'), '10 - 0', 'W10']
_t41 = [logo_absolute('41', 'Miami', 'left'), '4 - 6', 'W1']
_t42 = [logo_absolute('42', 'Houston', 'left'), '10 - 0', 'W10']
_t43 = [logo_absolute('43', 'Kansas City', 'left'), '9 - 1', 'W5']
_t44 = [logo_absolute('44', 'Los Angeles', 'left'), '7 - 3', 'L1']
_t45 = [logo_absolute('45', 'Los Angeles', 'left'), '5 - 5', 'W2']
_t46 = [logo_absolute('46', 'Milwaukee', 'left'), '10 - 0', 'W10']
_t47 = [logo_absolute('47', 'Minnesota', 'left'), '10 - 0', 'W10']
_t48 = [logo_absolute('48', 'New York', 'left'), '10 - 0', 'W10']
_t49 = [logo_absolute('49', 'New York', 'left'), '8 - 2', 'L1']
_t50 = [logo_absolute('50', 'Oakland', 'left'), '7 - 3', 'L3']
_t51 = [logo_absolute('51', 'Philadelphia', 'left'), '3 - 7', 'L5']
_t52 = [logo_absolute('52', 'Pittsburgh', 'left'), '6 - 4', 'L1']
_t53 = [logo_absolute('53', 'San Diego', 'left'), '9 - 1', 'W8']
_t54 = [logo_absolute('54', 'Seattle', 'left'), '10 - 0', 'W10']
_t55 = [logo_absolute('55', 'San Francisco', 'left'), '2 - 8', 'L4']
_t56 = [logo_absolute('56', 'St. Louis', 'left'), '10 - 0', 'W10']
_t57 = [logo_absolute('57', 'Tampa Bay', 'left'), '10 - 0', 'W10']
_t58 = [logo_absolute('58', 'Texas', 'left'), '9 - 1', 'W9']
_t59 = [logo_absolute('59', 'Toronto', 'left'), '9 - 1', 'W4']
_t60 = [logo_absolute('60', 'Washington', 'left'), '5 - 5', 'L1']

_breadcrumbs = [{
    'href': '/fairylab/',
    'name': 'Home'
}, {
    'href': '',
    'name': 'Exports'
}]

_tcols = [''] + [' class="text-center"'] * 5
_table = table(
    clazz='table-sm',
    hcols=_tcols,
    bcols=_tcols,
    body=[['AL East', 'BAL', 'BOS', 'NYY', 'TB', 'TOR'], [
        'AL Central',
        Exports._success('CWS'), 'CLE',
        Exports._success('DET'), 'KC',
        Exports._success('MIN')
    ], [
        'AL West', 'HOU',
        Exports._success('LAA'), 'OAK',
        Exports._success('SEA'),
        Exports._success('TEX')
    ], [
        'NL East',
        Exports._success('ATL'), 'MIA',
        Exports._success('NYM'), 'PHI',
        Exports._success('WAS')
    ], [
        'NL Central',
        Exports._success('CHC'),
        Exports._success('CIN'),
        Exports._success('MIL'), 'PIT',
        Exports._success('STL')
    ], [
        'NL West', 'ARI', 'COL',
        Exports._success('LAD'),
        Exports._success('SD'), 'SF'
    ]])

_breakdown = [Exports._success('16 new'), '14 old', Exports._secondary('0 ai')]
_info = 'Upcoming sim contains ' + ', '.join(_breakdown) + '.'
_live = card(title='53%', info=_info, table=_table, ts='30s ago')

_scols = [
    'class="position-relative"', ' class="text-center w-25"',
    ' class="text-center w-25"'
]
_standings = [
    table(
        clazz='border mt-3',
        hcols=_scols,
        bcols=_scols,
        head=['AL East', 'Last 10', 'Streak'],
        body=[_t34, _t48, _t57, _t59, _t33]),
    table(
        clazz='border mt-3',
        hcols=_scols,
        bcols=_scols,
        head=['AL Central', 'Last 10', 'Streak'],
        body=[_t40, _t47, _t43, _t35, _t38]),
    table(
        clazz='border mt-3',
        hcols=_scols,
        bcols=_scols,
        head=['AL West', 'Last 10', 'Streak'],
        body=[_t42, _t54, _t58, _t44, _t50]),
    table(
        clazz='border mt-3',
        hcols=_scols,
        bcols=_scols,
        head=['NL East', 'Last 10', 'Streak'],
        body=[_t32, _t49, _t60, _t41, _t51]),
    table(
        clazz='border mt-3',
        hcols=_scols,
        bcols=_scols,
        head=['NL Central', 'Last 10', 'Streak'],
        body=[_t37, _t46, _t56, _t36, _t52]),
    table(
        clazz='border mt-3',
        hcols=_scols,
        bcols=_scols,
        head=['NL West', 'Last 10', 'Streak'],
        body=[_t39, _t53, _t45, _t31, _t55])
]

subtitle = ''

tmpl = 'exports.html'

context = {
    'title': 'exports',
    'breadcrumbs': _breadcrumbs,
    'live': _live,
    'standings': _standings
}
