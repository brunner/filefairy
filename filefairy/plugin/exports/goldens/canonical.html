<!doctype html>
<html lang="en">

<head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
  <link rel="stylesheet"
        href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
        integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
        crossorigin="anonymous">
  <link rel="stylesheet"
        href="https://cdnjs.cloudflare.com/ajax/libs/open-iconic/1.1.1/font/css/open-iconic-bootstrap.min.css">
  <title>Fairylab | exports</title>
  <style>
    html {
      position: relative;
      min-height: 100%;
    }
    body {
      margin-bottom: 60px;
    }
    header {
      background-color: #f5f5f5;
      width: 100%;
    }
    footer {
      background-color: #f5f5f5;
      bottom: 0;
      height: 60px;
      line-height: 60px;
      position: absolute;
      width: 100%;
    }
    .alert, .form-control {
      padding: .75rem;
    }
    .alert .oi {
      line-height: 24px;
      padding-right: .75rem;
    }
    .anchor {
      padding-top: 0px;
      margin-top: 0px;
      transition: padding-top .35s ease, margin-top .35s ease;
    }
    .breadcrumb {
      background-color: inherit;
      line-height: 60px;
    }
    .flex-grow {
      flex-grow: 1;
    }
    .left-8p {
      left: 8px;
    }
    .navbar {
      padding: 0;
    }
    .navbar-nav {
      padding: .5rem 0;
    }
    .navbar .nav-link {
      padding: .5rem;
    }
    .pl-24p {
      padding-left: 24px;
    }
    .pr-24p {
      padding-right: 24px;
    }
    .right-8p {
      right: 8px;
    }
    .table, .table-sm {
      margin-bottom: 0;
      word-break: break-word;
    }
    .table .thead-light td {
      background-color: #f5f5f5;
    }
    .table-fixed {
      table-layout: fixed;
    }
    .table-sm td {
      border-top: 0;
      padding: 0 .3rem 0 0;
    }
    .table .td-sm {
      padding: .75rem .25rem;
    }
    .top-14p {
      top: 14px;
    }
    .w-10 {
      width: 10%;
    }
    .w-20 {
      width: 20%;
    }
    .w-40 {
      width: 40%;
    }
    .w-55p {
      width: 55px;
    }
    .w-105p {
      width: 105px;
    }
  </style>
</head>

<body>
  <header class="border-bottom sticky-top">
    <div class="container">
      <nav class="navbar navbar-light" aria-label="nav">
        <ol class="breadcrumb m-0 p-0">
          <li class="breadcrumb-item">
            <a href="/fairylab/">Home</a>
          </li>
          <li class="breadcrumb-item active"
              aria-current="page">Exports</li>
        </ol>
          <button class="navbar-toggler border-0" type="button"
                data-toggle="collapse" data-target="#nav-dropdown"
                aria-controls="#nav-dropdown" aria-expanded="false"
                aria-label="nav toggler">
          <span class="navbar-toggler-icon"></span>
        </button>
        <div class="collapse navbar-collapse border-top" id="nav-dropdown">
          <ul class="navbar-nav">
            <li class="nav-item">
              <a class="nav-link" href="#tracker">Tracker</a>
            </li>
            <li class="nav-item">
              <a class="nav-link" href="#recorded">Recorded</a>
            </li>
          </ul>
        </div>
      </nav>
    </div>
  </header>
  <main role="main" class="pb-3">
    <div class="container anchor" id="tracker">
      <div class="row">
        <div class="col-12">
          <div class="alert alert-secondary mt-3 mb-0">
            <span class="oi oi-cloudy text-secondary"></span>Tracker
          </div>
        </div>
      </div>
      <div class="row">
        <div class="col-12">
          <div class="card d-flex flex-column flex-grow mt-3">
            <div class="card-body d-flex flex-column">
              <h5 class="card-title">
                53%
              </h5>
              <p class="card-text flex-grow">Upcoming sim contains <span class="text-success border px-1">16 new</span>, 14 old, <span class="text-secondary">0 ai</span>.</p>
              <table class="table table-sm">
                <tbody>
                  <tr>
                    <td scope="row">AL East</td>
                    <td class="text-center">BAL</td>
                    <td class="text-center">BOS</td>
                    <td class="text-center">NYY</td>
                    <td class="text-center">TB</td>
                    <td class="text-center">TOR</td>
                  </tr>
                  <tr>
                    <td scope="row">AL Central</td>
                    <td class="text-center"><span class="text-success border px-1">CWS</span></td>
                    <td class="text-center">CLE</td>
                    <td class="text-center"><span class="text-success border px-1">DET</span></td>
                    <td class="text-center">KC</td>
                    <td class="text-center"><span class="text-success border px-1">MIN</span></td>
                  </tr>
                  <tr>
                    <td scope="row">AL West</td>
                    <td class="text-center">HOU</td>
                    <td class="text-center"><span class="text-success border px-1">LAA</span></td>
                    <td class="text-center">OAK</td>
                    <td class="text-center"><span class="text-success border px-1">SEA</span></td>
                    <td class="text-center"><span class="text-success border px-1">TEX</span></td>
                  </tr>
                  <tr>
                    <td scope="row">NL East</td>
                    <td class="text-center"><span class="text-success border px-1">ATL</span></td>
                    <td class="text-center">MIA</td>
                    <td class="text-center"><span class="text-success border px-1">NYM</span></td>
                    <td class="text-center">PHI</td>
                    <td class="text-center"><span class="text-success border px-1">WAS</span></td>
                  </tr>
                  <tr>
                    <td scope="row">NL Central</td>
                    <td class="text-center"><span class="text-success border px-1">CHC</span></td>
                    <td class="text-center"><span class="text-success border px-1">CIN</span></td>
                    <td class="text-center"><span class="text-success border px-1">MIL</span></td>
                    <td class="text-center">PIT</td>
                    <td class="text-center"><span class="text-success border px-1">STL</span></td>
                  </tr>
                  <tr>
                    <td scope="row">NL West</td>
                    <td class="text-center">ARI</td>
                    <td class="text-center">COL</td>
                    <td class="text-center"><span class="text-success border px-1">LAD</span></td>
                    <td class="text-center"><span class="text-success border px-1">SD</span></td>
                    <td class="text-center">SF</td>
                  </tr>
                </tbody>
              </table>
            </div>
            <div class="card-footer text-muted text-truncate">
              <small>Last activity: 30s ago</small>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div class="container anchor" id="recorded">
      <div class="row">
        <div class="col-12">
          <div class="alert alert-secondary mt-3 mb-0">
            <span class="oi oi-target text-secondary"></span>Recorded
          </div>
        </div>
      </div>
      <div class="row">
        <div class="col-md-6">
          <table class="table border mt-3">
            <thead class="thead-light">
              <tr>
                <td scope="col"class="position-relative">AL East</td>
                <td scope="col" class="text-center w-25">Last 10</td>
                <td scope="col" class="text-center w-25">Streak</td>
              </tr>
            </thead>
            <tbody>
              <tr>
                <td scope="row"class="position-relative"><img src="https://orangeandblueleaguebaseball.com/StatsLab/reports/news/html/images/team_logos/boston_red_sox_40.png" width="20" height="20" border="0" class="position-absolute left-8p top-14p"><span class="align-middle d-block text-truncate pl-24p">Boston</span></td>
                <td class="text-center w-25">10 - 0</td>
                <td class="text-center w-25">W10</td>
              </tr>
              <tr>
                <td scope="row"class="position-relative"><img src="https://orangeandblueleaguebaseball.com/StatsLab/reports/news/html/images/team_logos/new_york_yankees_40.png" width="20" height="20" border="0" class="position-absolute left-8p top-14p"><span class="align-middle d-block text-truncate pl-24p">New York</span></td>
                <td class="text-center w-25">10 - 0</td>
                <td class="text-center w-25">W10</td>
              </tr>
              <tr>
                <td scope="row"class="position-relative"><img src="https://orangeandblueleaguebaseball.com/StatsLab/reports/news/html/images/team_logos/tampa_bay_rays_40.png" width="20" height="20" border="0" class="position-absolute left-8p top-14p"><span class="align-middle d-block text-truncate pl-24p">Tampa Bay</span></td>
                <td class="text-center w-25">10 - 0</td>
                <td class="text-center w-25">W10</td>
              </tr>
              <tr>
                <td scope="row"class="position-relative"><img src="https://orangeandblueleaguebaseball.com/StatsLab/reports/news/html/images/team_logos/toronto_blue_jays_40.png" width="20" height="20" border="0" class="position-absolute left-8p top-14p"><span class="align-middle d-block text-truncate pl-24p">Toronto</span></td>
                <td class="text-center w-25">9 - 1</td>
                <td class="text-center w-25">W4</td>
              </tr>
              <tr>
                <td scope="row"class="position-relative"><img src="https://orangeandblueleaguebaseball.com/StatsLab/reports/news/html/images/team_logos/baltimore_orioles_40.png" width="20" height="20" border="0" class="position-absolute left-8p top-14p"><span class="align-middle d-block text-truncate pl-24p">Baltimore</span></td>
                <td class="text-center w-25">6 - 4</td>
                <td class="text-center w-25">W1</td>
              </tr>
            </tbody>
          </table>
          <table class="table border mt-3">
            <thead class="thead-light">
              <tr>
                <td scope="col"class="position-relative">AL Central</td>
                <td scope="col" class="text-center w-25">Last 10</td>
                <td scope="col" class="text-center w-25">Streak</td>
              </tr>
            </thead>
            <tbody>
              <tr>
                <td scope="row"class="position-relative"><img src="https://orangeandblueleaguebaseball.com/StatsLab/reports/news/html/images/team_logos/detroit_tigers_40.png" width="20" height="20" border="0" class="position-absolute left-8p top-14p"><span class="align-middle d-block text-truncate pl-24p">Detroit</span></td>
                <td class="text-center w-25">10 - 0</td>
                <td class="text-center w-25">W10</td>
              </tr>
              <tr>
                <td scope="row"class="position-relative"><img src="https://orangeandblueleaguebaseball.com/StatsLab/reports/news/html/images/team_logos/minnesota_twins_40.png" width="20" height="20" border="0" class="position-absolute left-8p top-14p"><span class="align-middle d-block text-truncate pl-24p">Minnesota</span></td>
                <td class="text-center w-25">10 - 0</td>
                <td class="text-center w-25">W10</td>
              </tr>
              <tr>
                <td scope="row"class="position-relative"><img src="https://orangeandblueleaguebaseball.com/StatsLab/reports/news/html/images/team_logos/kansas_city_royals_40.png" width="20" height="20" border="0" class="position-absolute left-8p top-14p"><span class="align-middle d-block text-truncate pl-24p">Kansas City</span></td>
                <td class="text-center w-25">9 - 1</td>
                <td class="text-center w-25">W5</td>
              </tr>
              <tr>
                <td scope="row"class="position-relative"><img src="https://orangeandblueleaguebaseball.com/StatsLab/reports/news/html/images/team_logos/chicago_white_sox_40.png" width="20" height="20" border="0" class="position-absolute left-8p top-14p"><span class="align-middle d-block text-truncate pl-24p">Chicago</span></td>
                <td class="text-center w-25">9 - 1</td>
                <td class="text-center w-25">L1</td>
              </tr>
              <tr>
                <td scope="row"class="position-relative"><img src="https://orangeandblueleaguebaseball.com/StatsLab/reports/news/html/images/team_logos/cleveland_indians_40.png" width="20" height="20" border="0" class="position-absolute left-8p top-14p"><span class="align-middle d-block text-truncate pl-24p">Cleveland</span></td>
                <td class="text-center w-25">8 - 2</td>
                <td class="text-center w-25">W2</td>
              </tr>
            </tbody>
          </table>
          <table class="table border mt-3">
            <thead class="thead-light">
              <tr>
                <td scope="col"class="position-relative">AL West</td>
                <td scope="col" class="text-center w-25">Last 10</td>
                <td scope="col" class="text-center w-25">Streak</td>
              </tr>
            </thead>
            <tbody>
              <tr>
                <td scope="row"class="position-relative"><img src="https://orangeandblueleaguebaseball.com/StatsLab/reports/news/html/images/team_logos/houston_astros_40.png" width="20" height="20" border="0" class="position-absolute left-8p top-14p"><span class="align-middle d-block text-truncate pl-24p">Houston</span></td>
                <td class="text-center w-25">10 - 0</td>
                <td class="text-center w-25">W10</td>
              </tr>
              <tr>
                <td scope="row"class="position-relative"><img src="https://orangeandblueleaguebaseball.com/StatsLab/reports/news/html/images/team_logos/seattle_mariners_40.png" width="20" height="20" border="0" class="position-absolute left-8p top-14p"><span class="align-middle d-block text-truncate pl-24p">Seattle</span></td>
                <td class="text-center w-25">10 - 0</td>
                <td class="text-center w-25">W10</td>
              </tr>
              <tr>
                <td scope="row"class="position-relative"><img src="https://orangeandblueleaguebaseball.com/StatsLab/reports/news/html/images/team_logos/texas_rangers_40.png" width="20" height="20" border="0" class="position-absolute left-8p top-14p"><span class="align-middle d-block text-truncate pl-24p">Texas</span></td>
                <td class="text-center w-25">9 - 1</td>
                <td class="text-center w-25">W9</td>
              </tr>
              <tr>
                <td scope="row"class="position-relative"><img src="https://orangeandblueleaguebaseball.com/StatsLab/reports/news/html/images/team_logos/los_angeles_angels_40.png" width="20" height="20" border="0" class="position-absolute left-8p top-14p"><span class="align-middle d-block text-truncate pl-24p">Los Angeles</span></td>
                <td class="text-center w-25">7 - 3</td>
                <td class="text-center w-25">L1</td>
              </tr>
              <tr>
                <td scope="row"class="position-relative"><img src="https://orangeandblueleaguebaseball.com/StatsLab/reports/news/html/images/team_logos/oakland_athletics_40.png" width="20" height="20" border="0" class="position-absolute left-8p top-14p"><span class="align-middle d-block text-truncate pl-24p">Oakland</span></td>
                <td class="text-center w-25">7 - 3</td>
                <td class="text-center w-25">L3</td>
              </tr>
            </tbody>
          </table>
        </div>
        <div class="col-md-6">
          <table class="table border mt-3">
            <thead class="thead-light">
              <tr>
                <td scope="col"class="position-relative">NL East</td>
                <td scope="col" class="text-center w-25">Last 10</td>
                <td scope="col" class="text-center w-25">Streak</td>
              </tr>
            </thead>
            <tbody>
              <tr>
                <td scope="row"class="position-relative"><img src="https://orangeandblueleaguebaseball.com/StatsLab/reports/news/html/images/team_logos/atlanta_braves_40.png" width="20" height="20" border="0" class="position-absolute left-8p top-14p"><span class="align-middle d-block text-truncate pl-24p">Atlanta</span></td>
                <td class="text-center w-25">10 - 0</td>
                <td class="text-center w-25">W10</td>
              </tr>
              <tr>
                <td scope="row"class="position-relative"><img src="https://orangeandblueleaguebaseball.com/StatsLab/reports/news/html/images/team_logos/new_york_mets_40.png" width="20" height="20" border="0" class="position-absolute left-8p top-14p"><span class="align-middle d-block text-truncate pl-24p">New York</span></td>
                <td class="text-center w-25">8 - 2</td>
                <td class="text-center w-25">L1</td>
              </tr>
              <tr>
                <td scope="row"class="position-relative"><img src="https://orangeandblueleaguebaseball.com/StatsLab/reports/news/html/images/team_logos/washington_nationals_40.png" width="20" height="20" border="0" class="position-absolute left-8p top-14p"><span class="align-middle d-block text-truncate pl-24p">Washington</span></td>
                <td class="text-center w-25">5 - 5</td>
                <td class="text-center w-25">L1</td>
              </tr>
              <tr>
                <td scope="row"class="position-relative"><img src="https://orangeandblueleaguebaseball.com/StatsLab/reports/news/html/images/team_logos/miami_marlins_40.png" width="20" height="20" border="0" class="position-absolute left-8p top-14p"><span class="align-middle d-block text-truncate pl-24p">Miami</span></td>
                <td class="text-center w-25">4 - 6</td>
                <td class="text-center w-25">W1</td>
              </tr>
              <tr>
                <td scope="row"class="position-relative"><img src="https://orangeandblueleaguebaseball.com/StatsLab/reports/news/html/images/team_logos/philadelphia_phillies_40.png" width="20" height="20" border="0" class="position-absolute left-8p top-14p"><span class="align-middle d-block text-truncate pl-24p">Philadelphia</span></td>
                <td class="text-center w-25">3 - 7</td>
                <td class="text-center w-25">L5</td>
              </tr>
            </tbody>
          </table>
          <table class="table border mt-3">
            <thead class="thead-light">
              <tr>
                <td scope="col"class="position-relative">NL Central</td>
                <td scope="col" class="text-center w-25">Last 10</td>
                <td scope="col" class="text-center w-25">Streak</td>
              </tr>
            </thead>
            <tbody>
              <tr>
                <td scope="row"class="position-relative"><img src="https://orangeandblueleaguebaseball.com/StatsLab/reports/news/html/images/team_logos/cincinnati_reds_40.png" width="20" height="20" border="0" class="position-absolute left-8p top-14p"><span class="align-middle d-block text-truncate pl-24p">Cincinnati</span></td>
                <td class="text-center w-25">10 - 0</td>
                <td class="text-center w-25">W10</td>
              </tr>
              <tr>
                <td scope="row"class="position-relative"><img src="https://orangeandblueleaguebaseball.com/StatsLab/reports/news/html/images/team_logos/milwaukee_brewers_40.png" width="20" height="20" border="0" class="position-absolute left-8p top-14p"><span class="align-middle d-block text-truncate pl-24p">Milwaukee</span></td>
                <td class="text-center w-25">10 - 0</td>
                <td class="text-center w-25">W10</td>
              </tr>
              <tr>
                <td scope="row"class="position-relative"><img src="https://orangeandblueleaguebaseball.com/StatsLab/reports/news/html/images/team_logos/st_louis_cardinals_40.png" width="20" height="20" border="0" class="position-absolute left-8p top-14p"><span class="align-middle d-block text-truncate pl-24p">St. Louis</span></td>
                <td class="text-center w-25">10 - 0</td>
                <td class="text-center w-25">W10</td>
              </tr>
              <tr>
                <td scope="row"class="position-relative"><img src="https://orangeandblueleaguebaseball.com/StatsLab/reports/news/html/images/team_logos/chicago_cubs_40.png" width="20" height="20" border="0" class="position-absolute left-8p top-14p"><span class="align-middle d-block text-truncate pl-24p">Chicago</span></td>
                <td class="text-center w-25">8 - 2</td>
                <td class="text-center w-25">L1</td>
              </tr>
              <tr>
                <td scope="row"class="position-relative"><img src="https://orangeandblueleaguebaseball.com/StatsLab/reports/news/html/images/team_logos/pittsburgh_pirates_40.png" width="20" height="20" border="0" class="position-absolute left-8p top-14p"><span class="align-middle d-block text-truncate pl-24p">Pittsburgh</span></td>
                <td class="text-center w-25">6 - 4</td>
                <td class="text-center w-25">L1</td>
              </tr>
            </tbody>
          </table>
          <table class="table border mt-3">
            <thead class="thead-light">
              <tr>
                <td scope="col"class="position-relative">NL West</td>
                <td scope="col" class="text-center w-25">Last 10</td>
                <td scope="col" class="text-center w-25">Streak</td>
              </tr>
            </thead>
            <tbody>
              <tr>
                <td scope="row"class="position-relative"><img src="https://orangeandblueleaguebaseball.com/StatsLab/reports/news/html/images/team_logos/colorado_rockies_40.png" width="20" height="20" border="0" class="position-absolute left-8p top-14p"><span class="align-middle d-block text-truncate pl-24p">Colorado</span></td>
                <td class="text-center w-25">10 - 0</td>
                <td class="text-center w-25">W10</td>
              </tr>
              <tr>
                <td scope="row"class="position-relative"><img src="https://orangeandblueleaguebaseball.com/StatsLab/reports/news/html/images/team_logos/san_diego_padres_40.png" width="20" height="20" border="0" class="position-absolute left-8p top-14p"><span class="align-middle d-block text-truncate pl-24p">San Diego</span></td>
                <td class="text-center w-25">9 - 1</td>
                <td class="text-center w-25">W8</td>
              </tr>
              <tr>
                <td scope="row"class="position-relative"><img src="https://orangeandblueleaguebaseball.com/StatsLab/reports/news/html/images/team_logos/los_angeles_dodgers_40.png" width="20" height="20" border="0" class="position-absolute left-8p top-14p"><span class="align-middle d-block text-truncate pl-24p">Los Angeles</span></td>
                <td class="text-center w-25">5 - 5</td>
                <td class="text-center w-25">W2</td>
              </tr>
              <tr>
                <td scope="row"class="position-relative"><img src="https://orangeandblueleaguebaseball.com/StatsLab/reports/news/html/images/team_logos/arizona_diamondbacks_40.png" width="20" height="20" border="0" class="position-absolute left-8p top-14p"><span class="align-middle d-block text-truncate pl-24p">Arizona</span></td>
                <td class="text-center w-25">5 - 5</td>
                <td class="text-center w-25">W1</td>
              </tr>
              <tr>
                <td scope="row"class="position-relative"><img src="https://orangeandblueleaguebaseball.com/StatsLab/reports/news/html/images/team_logos/san_francisco_giants_40.png" width="20" height="20" border="0" class="position-absolute left-8p top-14p"><span class="align-middle d-block text-truncate pl-24p">San Francisco</span></td>
                <td class="text-center w-25">2 - 8</td>
                <td class="text-center w-25">L4</td>
              </tr>
            </tbody>
          </table>
        </div>
      </div>
    </div>
  </main>
  <footer class="border-top">
    <div class="container text-center">
      <small class="text-muted">Last reloaded: 1985-10-26 06:02:30 PST</small>
    </div>
  </footer>

  <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
          integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN"
          crossorigin="anonymous"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js"
          integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q"
          crossorigin="anonymous"></script>
  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.3/js/bootstrap.min.js"
          integrity="sha384-a5N7Y/aK3qNeh15eJKGWxsqtnX/wWdSZSKp+81YjTmS15nvnvxKHuzaWwXHDli+4"
          crossorigin="anonymous"></script>
  <script>
    $('.navbar-toggler').one('click', function(e) {
      setTimeout(function() {
        var height = $('header').height() + 1;
        var css = '<style> .anchor.offset { padding-top: %spx; margin-top: -%spx; } </style>';
        $(css.replace(/%s/g, height)).appendTo('head');
      }, 500);
    });
    $('.navbar-toggler').on('click', function(e) {
      $('.anchor').toggleClass('offset');
    });
  </script>
</body>

</html>